src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
I'm still not convinced we should go with this, but I'll defer,

@nathanielmanistaatgoogle 
Do you have any thoughts on this?